Abcbadq/sjsu_shop
# SJSU-1007, Search feature

STORY:

Create a search function

front-end:

Search box in Navbar that can be accessed by guests as well as signed in users
create a new page to display results of the search (see SJSU-1010, maybe we can use that page for this)
back-end:

create algorithm that searches the database
reminder: the structure of a product object includes name, brand, tags

What I did:

-Add search bar and search button 
-Created search page to show result
-Wired to allow navigations between pages
-Created back-end that allow to search part of string ( case in-sensitive) 
-Created product model

To Verify:
_ Make sure you can navigate to other pages after the search page is showed
_ Uppercase and lowercase search key should give the same results
